docs/source/iroha_2_whitepaper.md

```diff
@@ -1,4 +1,4 @@
-# Iroha v2.0 
```

Comment:
I want to conclude that I'm not qualified to be a reviewer for now. Let me decline my review. I'd like to contribute by adding tests later